eng/pipelines/templates/stages/platform-matrix-quick.json

```diff
@@ -0,0 +1,22 @@
+{ 
+    "displayNames": { 
+        "_": ""
+    }, 
+    "include": [
+          {
+            "StaticConfigs": {
+              "Ubuntu22": {
+                "OSVmImage": "MMSUbuntu22.04",
```

Comment:
Do you also want to remove this from the default matrix?

Reply:
It is removed from the other build matrix: https://github.com/Azure/azure-sdk-for-cpp/pull/4522/files#diff-6d57a4119ae22b0fede4d3aab5dff565cee910416d4eaa647d83e1d749c9b559L159

Reply:
Perhaps I'm missing something but that just shows removal of `included: {}` how does that map to this build configuration?